packages/dsi-logo-maker/functions/textAndImageCanvas/index.js

```diff
@@ -21,9 +23,10 @@ export const handler = ({ inputs, mechanic }) => {
   const words = text.split(" ").map((s) => s.toUpperCase());
   const colors = getColors("Custom Colors", null, colorsString.split(","));
   const height = Math.floor((width / ratio) * rows);
+  const font = await loadOpentypeFont(fontMode);
 
   const blockGeometry = computeBlockGeometry(width, height, rows, cols);
-  const baseBricks = computeBaseBricks(words, blockGeometry.fontSize);
+  const baseBricks = computeBaseBricks(words, blockGeometry.fontSize, font);
```

Comment:
This is probably unrelated to your changes, but the image input does not work for me on this one.

Reply:
Hmmm, weird. It was working for me when I tested it!